app/models/chargeback.rb

```diff
@@ -72,6 +72,13 @@ def self.build_results_for_report_chargeback(options)
     elsif options[:tag]
       vms = Vm.find_tagged_with(:all => options[:tag], :ns => "*")
       vms &= report_user.accessible_vms if report_user && report_user.self_service?
+    elsif options[:tenant]
+      tenant = Tenant.find_by(:id => options[:tenant])
```

Comment:
@aljesusg 
`tenant = Tenant.find_by(:id => options[:tenant])` can be `tenant = Tenant.find_by(tenant = Tenant.find_by(options[:tenant])`

and `options[:tenant]` is Tenant#id right ? so can you use (and store into db) `:tenant_id` rather ?
maybe this will need to change it at place where `tenant_id` is stored into miq_report table also